server/index.js

```diff
@@ -1,14 +1,18 @@
 const express = require('express');
 const db = require('../database/index.js');
 const axios = require('axios');
+ const cors = require('cors');
 const bodyParser = require('body-parser');
 const app = express();
 const port = 3001;
 
+app.use(cors());
 app.use(express.static(__dirname + '/../client/public'));
 //app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: true}));
 
+//const origins = ['http://localhost:3000'];
```

Comment:
did you find that you didnt have to do anything additional after just adding CORS to your server/app? interesting...

Reply:
yeah.. not sure if this poses any security issues since i think it's allowing all requests access...